deployment/aws/core_backend/bootstrap.sh

```diff
@@ -42,10 +42,6 @@ export USER1_USERNAME=$(get_secret_value ${SECRET_USER_CREDENTIALS} "user1_usern
 export USER1_PASSWORD=$(get_secret_value ${SECRET_USER_CREDENTIALS} "user1_password" "json")
```

Comment:
So in secrets, we need to change USER1 to be Admin?
A future cleanup is to just add it ADMIN_USERNAME everywhere but ok for now.

Reply:
Yeah if we want, but doesn't matter what it is really. We should do that in a future cleanup for clarity.